scipy/special/_add_newdocs.py

```diff
@@ -696,17 +696,93 @@ def add_newdoc(name, doc):
            Functions, Inverses, and Other Parameters.
     """)
 
-add_newdoc("binom",
-    """
-    binom(n, k)
+add_newdoc(
+    "binom",
+    r"""
+    binom(x, y)
+
+    Binomial coefficient considered as a function of two real variables.
+
+    For real arguments, the binomial coefficient is defined as
+
+    .. math::
+
+        \binom{x}{y} = \frac{\Gamma(x + 1)}{\Gamma(y + 1)\Gamma(x - y + 1)} =
+            \frac{1}{(x + 1)\mathrm{B}(x - y + 1, y + 1)}
+
+    Where :math:`\Gamma` is the Gamma function and :math:`\mathrm{B}` is
+    the Beta function [1]_. See `gamma` and `beta`.
```

Comment:
```suggestion
    Where :math:`\Gamma` is the Gamma function (`gamma`) and :math:`\mathrm{B}`
    is the Beta function (`beta`) [1]_.
```

Nitpick; I think it looks better this way but feel free to ignore if you don't agree.